ci/docker/image-builder/src/integTest/java/ru/avito/image_builder/internal/CliDockerTest.kt

```diff
@@ -0,0 +1,86 @@
+package ru.avito.image_builder.internal
```

Comment:
I don't get what are you testing there

Reply:
Tests check Docker client implementation.
It uses CLI API. The weak point is passing arguments and parsing output.
Thus, I want to be sure at least in basic scenarios.